draft-ietf-ppm-dap.md

```diff
@@ -575,24 +573,28 @@ particularly important for controlling the amount of noise added to reports
 by Clients (or added to aggregate shares by Aggregators) in order to achieve
 differential privacy.
 
-To get the aggregate of the next batch, the Collector issues a query of type
-`next-batch`. (See FixedSizeQuery in {{query}}.) The response to this query (see
-{{collect-flow}}) includes a "batch ID", which can be used to query the same
-batch later on. This is done using the `by-batch-id` batch-query type.
-
 For this query type, the Aggregators group reports into arbitrary batches such
-that each batch has roughly the same number of reports. In addition to the
-minumum batch size common to all query types, the conifugration includes a
-"maximum batch size", `max_batch_size`, that determines maximum number of
-reports per batch.
+that each batch has roughly the same number of reports. These batches are
+identified by opaque "batch IDs", allocated in an arbitrary fashion by the
+Leader.
+
+To get the aggregate of a batch, the Collector issues a query
+specifying the batch ID of interest (see `FixedSizeQuery` in {{query}}). The
+Collector may not know which batch ID it is interested in; in this case, it can
```

Comment:
If the batchID is allocated by Leader in an arbitrary fashion, then wouldn't the collect HAVE TO issue a current-batch query first? Or are we deliberately not being specific about the way current-batch is used, so some OOB agreement can be used to "predict" the batchID?

Reply:
Typically yes, I expect that the Collector would use `current-batch` first (outside of OOB agreement you mention), and then possibly use one or more `by-batch-id` queries for that batch, depending on the VDAF.

I was writing this text as an explanation of available functionality/expectations for the `fixed-size` query type, rather than a step-by-step guide to how the query types are expected to be used. I can change if folks like?